sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/Azure.ResourceManager.DesktopVirtualization.sln

```diff
@@ -1,19 +1,17 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30309.148
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32421.90
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{96C8BA92-F974-4A9A-884D-7066DB2B9811}") = "Azure.ResourceManager.DesktopVirtualization", "src\Azure.ResourceManager.DesktopVirtualization.csproj", "{DDEE3A0C-9F4E-4CA0-ADF5-A511881AC189}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.DesktopVirtualization", "src\Azure.ResourceManager.DesktopVirtualization.csproj", "{DDEE3A0C-9F4E-4CA0-ADF5-A511881AC189}"
 EndProject
-Project("{96C8BA92-F974-4A9A-884D-7066DB2B9811}") = "Azure.ResourceManager.DesktopVirtualization.Tests", "tests\Azure.ResourceManager.DesktopVirtualization.Tests.csproj", "{E36A8647-6C66-4C6B-81DE-C8D633DBB8B0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.DesktopVirtualization.Tests", "tests\Azure.ResourceManager.DesktopVirtualization.Tests.csproj", "{E36A8647-6C66-4C6B-81DE-C8D633DBB8B0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager", "..\..\resourcemanager\Azure.ResourceManager\src\Azure.ResourceManager.csproj", "{AE4E1BDC-D26A-415E-B56C-E9B98BCC66CA}"
```

Comment:
Please remove the project `Azure.ResourceManager` and `Azure.Core.TestFramework`, we don't need them here, the refernence has been added. For example, [here](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.targets#L227) is the test framework dependency.